packages/plugin-client-ip/src/client-ip.ts

```diff
@@ -0,0 +1,32 @@
+import { Plugin } from 'packages/core/types'
+import assign from '@bugsnag/core/lib/es-utils/assign'
+
+/*
+ * Prevent collection of user IPs
+ */
+const plugin: Plugin = {
+  load: client => {
+    // @ts-expect-error _config is private API
```

Comment:
could we extend the `Plugin` interface here to get rid of these instances of ts-expect-error?